packages/web/src/components/Contributions/ContributionsBox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Link } from '@chakra-ui/react';
+
+export interface ContributionsBoxProps {
+  cbox: {
+    id: string;
+    nodeID: string;
+    type: string;
+    score: number;
+    contributedAt: Date;
+    description: string | null;
+    url: string;
+  };
+}
+
+export const ContributionsBox: React.FC<ContributionsBoxProps> = ({ cbox }) => {
```

Comment:
After the change above, here's how you can get your variables:
```suggestion
export const ContributionsBox: React.FC<ContributionsBoxProps> = ({ id, nodeID, type, score, contributedAt, description, url }) => {
```